modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -859,7 +859,7 @@ module.exports = (self) => {
       if (one[field.name].length !== two[field.name].length) {
         return false;
       }
-      for (let i = 0; (i < one.length); i++) {
+      for (let i = 0; (i < one[field.name].length); i++) {
```

Comment:
Fixes isEqual method that tried to iterate over a doc object length which is null.